api/src/commonTest/kotlin/com.buenaflor.kflogger/util/IgnoreIos.kt

```diff
@@ -0,0 +1,3 @@
+package com.buenaflor.kflogger.util
+
+@Target(AnnotationTarget.CLASS, AnnotationTarget.FUNCTION) expect annotation class IgnoreIos()
```

Comment:
Curious to know more about this annotation class -- what's the intention behind this?

Reply:
Since `iosMain` actuals only consist of `TODO("...")` running tests on that platform will always fail. `IgnoreIos` can be temporarily used to skip iOS tests - at least until we actually implement the iOS specifics